README.md

```diff
@@ -119,6 +119,12 @@ $ make cluster/install/local
 $ make code/run
 ```
 
+# Generating CSV
+
+1. Update the `AMO_VERSION` and `PREV_AMO_VERSION` values in [Makefile](https://github.com/integr8ly/application-monitoring-operator/blob/master/Makefile#L17)
+2. Run `make gen/csv` to generate a new manifest. 
+3. Add `clusterPermissions` block to new `application-monitoring-operator.vX.X.X.clusterserviceversion.yaml` 
```

Comment:
To be more specific you could say something like:
ensure `clusterPermissions` block in the generated csv is up to date with the cluster roles in the [`cluster-roles`](./deploy/cluster-roles) directory